server/channels/db/migrations/postgres/000126_scheduled_posts_table.up.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE IF NOT EXISTS scheduledposts (
```

Comment:
Are we sure no other indices are required? There seems to be joins with some other tables where indices would be helpful.

Reply:
I'll add all the required indexes once all the store functions are done. Still a bit early to know the exact queries needed.

Reply:
Created https://mattermost.atlassian.net/browse/MM-60118 to track the same.

Reply:
Oh okay, so this PR is still in draft stage? Feel free to request a review once it's ready. Thanks.

Reply:
This PR is NOT in draft. This PR ios against a feature brancvh, not master. I'll keep working off the feature branch, adding stuff in stages and merging into feature branch. So, not every PR has to be a complete PR. For example, this PR can be merged into feature branch, adn I can add those indexes one month from now and we would still be good.

Reply:
Oh my bad. Didn't realize this was a feature branch. In that case, you were right to remove my review. Please add me in the final PR that merges to master. Thanks.